plugins/modules/device_configs_backup_workflow_manager.py

```
@@ -1213,57 +1623,82 @@ def get_want(self, config):
             It validates the IP address list and file password, generates a new password if none is provided,
             retrieves device IDs, and updates the desired state with necessary parameters.
         """
+        self.log("Starting the process to prepare the desired state (want) based on the provided configuration.", "INFO")
+
         self.want = {}
 
         # Retrieve configuration parameters
         file_path = config.get("file_path")
         file_password = config.get("file_password")
         ip_address_list = config.get("ip_address_list")
+        file_types = config.get("config_file_types")
+        if file_types:
+            current_version = self.get_ccc_version()
+            self.log(f"Current Catalyst Center Version: {current_version}", "DEBUG")
+            if self.compare_dnac_versions(current_version, "2.3.7.9") < 0:
+                self.fail_and_exit(
+                    "The 'config_file_types' parameter is not supported for Catalyst Center version 2.3.7.6. "
+                    "It is supported from version 2.3.7.9 onwards."
+                )
```

Why 2.3.7.6?

Should be current_version version instead of 2.3.7.6.

---

I Added a negative test scenario. If the parameter is used while testing with version 2.3.7.6(old_version), it should throw this error.